geode-for-redis/src/distributedTest/java/org/apache/geode/redis/internal/executor/auth/AuthWhileServersRestartDUnitTest.java

```diff
@@ -50,6 +50,7 @@
   @ClassRule
   public static ExecutorServiceRule executor = new ExecutorServiceRule();
 
+  private static MemberVM server1;
```

Comment:
Sorry, where is this variable used? I see it being set, but I don't see where the value gets used.

Reply:
Thanks. I was using it but that changed... It's gone now.